openlibrary/catalog/add_book/__init__.py

```diff
@@ -678,6 +685,189 @@ def load_data(rec, account_key=None):
     return reply
 
 
+def update_matched_edition(
+    edition: Edition, rec: dict, need_edition_save: bool, account_key: str | None = None
+) -> tuple[Edition, bool]:
+    """
+    Update `edition` fields to add values from `rec` that don't currently
+    exist on `edition`. The default is to prefer existing values rather than
```

Comment:
That comment is currently misleading, but hopefully be the close of the weekend it will no longer be so. I want to make it so if certain types of records are identified of being particularly low quality, that there is the option to overwrite them with *extreme prejudice* using MARC data.

Reply:
This is no longer misleading. :D